src/build.sh

```diff
@@ -353,6 +366,10 @@ function parse_build_options()
         options_values['FULL_CLEANUP']=1
         shift
         ;;
+      --verbose | -v)
+        options_values['VERBOSE']=1
+        shift
```

Comment:
Add a simple unit test for this new parameter (same idea for all the commits in this PR).